client/view/Form/Form.js

```diff
@@ -46,31 +44,34 @@ export function submitForm() {
 }
 
 function onNextChange(cb) {
-  textarea.addEventListener('input', cb, { once: true })
+  form.addEventListener('submit', cb, { once: true })
 }
 
 function renderError(message) {
   errorMessage.innerHTML = message
-  form.classList.add('is-error')
+  errorMessage.hidden = false
   textarea.setAttribute('aria-errormessage', 'form_error')
   textarea.setAttribute('aria-invalid', 'true')
   onNextChange(() => {
     textarea.removeAttribute('aria-errormessage')
     textarea.removeAttribute('aria-invalid')
     errorMessage.innerHTML = ''
-    form.classList.remove('is-error')
+    errorMessage.hidden = true
   })
 }
 
 function renderWarning(message) {
   warningMessage.innerHTML = message
-  form.classList.add('is-warning')
+  warningMessage.hidden = false
   onNextChange(() => {
     warningMessage.innerHTML = ''
-    form.classList.remove('is-warning')
+    warningMessage.hidden = true
   })
 }
 
+let formatHintText = text =>
+  `<p>${text.replace(/`([^`]+)`/g, '<strong>$1</strong>')}</p>`
```

Comment:
We need to have multiple blocks for multiple warnings, not a single block with `<p>`.

Reply:
it's ok?)

![image](https://user-images.githubusercontent.com/22644149/189086550-6678303a-4679-4dcb-b0f8-8619414eb292.png)

Reply:
Yes!

Reply:
Done

Additional changes:
- Rename. `Hint` -> `Alert`
- Moved to `Alert.js`
- Create callback `onFormSubmit` in `Form.js`

Reply:
Links to test-cases

**1 warning**
https://preview-482---browserslist-vw3nqlhdtq-ue.a.run.app/#q=%3E%3D+5%25+in+US

**3 warnings**
https://preview-482---browserslist-vw3nqlhdtq-ue.a.run.app/#q=last+2+Chrome+versions

**Error**
https://preview-482---browserslist-vw3nqlhdtq-ue.a.run.app/#q=%3E%3E

**Normal**
https://preview-482---browserslist-vw3nqlhdtq-ue.a.run.app/#q=defaults